domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/Attribution.java

```diff
@@ -0,0 +1,383 @@
+         */
+        public AttributionBuilder isOperator(@Nullable final Integer isOperator) {
+            if (isOperator != null) {
+                this.isOperator = Objects.equals(isOperator, 1);
+            }
+            this.originalIsOperatorInteger = isOperator;
+            return this;
+        }
+
+        /**
+         * Sets field attributionUrl value and returns this
+         *
+         * @param attributionUrl URL of the organization
+         * @return builder for future object creation
+         */
+        public AttributionBuilder attributionUrl(@Nullable final String attributionUrl) {
+            this.attributionUrl = attributionUrl;
+            return this;
+        }
+
+        /**
+         * Sets field attributionEmail value and returns this
+         *
+         * @param attributionEmail email of the organization
+         * @return builder for future object creation
+         */
+        public AttributionBuilder attributionEmail(@Nullable final String attributionEmail) {
+            this.attributionEmail = attributionEmail;
+            return this;
+        }
+
+        /**
+         * Sets field attributionPhone value and returns this
+         *
+         * @param attributionPhone phone number of the organization
+         * @return builder for future object creation
+         */
+        public AttributionBuilder attributionPhone(@Nullable final String attributionPhone) {
+            this.attributionPhone = attributionPhone;
+            return this;
+        }
+
+
+        /**
+         * Returns {@code EntityBuildResult} representing a row from attributions.txt if the requirements from the
+         * official GTFS specification are met. Otherwise, method returns a collection of notices specifying the issues.
+         *
+         * @return {@link EntityBuildResult} representing a row from attributions.txt if the requirements from the
+         * official GTFS specification are met. Otherwise, method returns a collection of notices specifying the issues.
+         */
+        public EntityBuildResult<?> build() {
+            noticeCollection.clear();
+            if (organizationName == null ||
+                    (originalIsOperatorInteger != null &&
+                            (originalIsOperatorInteger < 0 || originalIsOperatorInteger > 1)) ||
+                    (originalIsAuthorityInteger != null &&
+                            (originalIsAuthorityInteger < 0 || originalIsAuthorityInteger > 1)) ||
+                    (originalIsProducerInteger != null &&
+                            (originalIsProducerInteger < 0 || originalIsProducerInteger > 1)) ||
+                    ((isAuthority == isProducer) && (isAuthority == isOperator) &&
+                            (originalIsProducerInteger == null || originalIsProducerInteger == 0))
+            ) {
+                final String entityId = attributionId + "; " + agencyId + "; " + routeId + "; " + tripId + "; " +
```

Comment:
Same as FareRule, see https://github.com/MobilityData/gtfs-validator/pull/72#discussion_r438366355 and https://github.com/MobilityData/gtfs-validator/pull/72#discussion_r438369254

Reply:
see https://github.com/MobilityData/gtfs-validator/pull/75#discussion_r441179373